modules/install/pages/non-root.adoc

```diff
@@ -250,9 +262,10 @@ To upgrade an existing non-root Couchbase Server,
 Specify the new package to be used for upgrade; and specify the install location of the currently resident Couchbase Server.
 For example:
 +
+[source, console, subs=+quotes]
 ----
-./cb-non-package-installer --upgrade --install-location ./cb-install \
---package ./couchbase-server-enterprise-6.6.0-7854-centos7.x86_64.rpm
+./cb-non-package-installer-#<platform-suffix># --upgrade --install-location ./cb-install \
+--package ./couchbase-server-enterprise-6.6.0-7854-centos7.#<platform-suffix>#.rpm
```

Comment:
Since this is part of an upgrade example (line 268), let's use 7.0.0 (instead of 7.1.0).
Instead of 
`--package ./couchbase-server-enterprise-6.6.0-7854-centos7.#<platform-suffix>#.rpm`
Use
`--package ./couchbase-server-enterprise-7.0.0-amzn2.#<platform-suffix>#.rpm`